cpp/include/cudf/partitioning.hpp

```diff
@@ -17,6 +17,8 @@
 #pragma once
 
 #include <cudf/types.hpp>
+#include <rmm/cuda_stream_view.hpp>
+
```

Comment:
Group headers by library, with a blank line between them. clang-format will sort within groups. 
```suggestion
#include <cudf/types.hpp>

#include <rmm/cuda_stream_view.hpp>

```